raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -65,18 +65,18 @@ contract MonitoringService is Utils {
     )
         public
     {
-        require(_token_address != address(0x0));
-        require(_service_registry_address != address(0x0));
-        require(_udc_address != address(0x0));
-        require(contractExists(_token_address));
-        require(contractExists(_service_registry_address));
-        require(contractExists(_udc_address));
+        require(_token_address != address(0x0), "Token at address zero?");
```

Comment:
why use question marks here? below you don't use them: https://github.com/raiden-network/raiden-contracts/pull/1115/files#diff-75ab51d9425251e3761a502084d14fd7R312

Reply:
I'll make it consistent.